bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/SonosXMLParser.java

```diff
@@ -788,46 +801,75 @@ public SonosMetaData getMetaData() {
 
     private static class RenderingControlEventHandler extends DefaultHandler {
 
-        private final Map<String, String> changes = new HashMap<>();
+        private final Map<String, @Nullable String> changes = new HashMap<>();
 
         private boolean getPresetName = false;
-        private String presetName;
+        private @Nullable String presetName;
 
         @Override
-        public void startElement(String uri, String localName, String qName, Attributes atts) throws SAXException {
+        public void startElement(@Nullable String uri, @Nullable String localName, @Nullable String qName,
+                @Nullable Attributes attributes) throws SAXException {
+            if (qName == null) {
+                return;
+            }
             if ("Volume".equals(qName)) {
```

Comment:
If you turn this into a switch statement you could merge the "Volume","Mute", and "Loudness" cases since they have the same logic.